doc/schemas/node_openapi.json

```diff
@@ -987,6 +996,24 @@
     },
     "/node/commit": {
       "get": {
+        "parameters": [
+          {
+            "in": "query",
+            "name": "view_history",
+            "required": true,
+            "schema": {
+              "$ref": "#/components/schemas/boolean"
+            }
+          },
+          {
+            "in": "query",
+            "name": "view_history_since",
```

Comment:
Perhaps arguably better called "view_history_since_seqno" or "from_seqno", since it takes a single integer, which could equally be a seqno or a view?

Reply:
Hah, it was view, and "view_history_since_view" does seem a bit silly.